packages/network/src/connection/webrtc/WebRtcConnection.ts

```diff
@@ -29,6 +29,12 @@ export interface ConstructorOptions {
     flushRetryTimeout?: number
     messageQueue: MessageQueue<string>
     deferredConnectionAttempt: DeferredConnectionAttempt
+    portRange: WebRtcPortRange
+}
+
+export interface WebRtcPortRange {
+    begin: number
+    end: number
 }
```

Comment:
I'd rename the fields to min and max so that we are consistent with our naming. 

Typically when we have some kind of range with upper and/or lower bounds, we name fields using min and max prefixes. E.g. `minTimestamp`+`maxTimestamp`, `minPropagationTargets`, `maxMessages`, `maxNeighborsPerNode`...

```suggestion
export interface WebRtcPortRange {
    min: number
    max: number
}
```